port/osx/omrintrospect.c

```diff
@@ -353,9 +441,22 @@ omrintrospect_threads_startDo_with_signal(struct OMRPortLibrary *portLibrary, J9
 	data->state = state;
 	data->filterThread = mach_thread_self();
 
+	/* Create a pipe to allow the resumed thread to report it has completed
+	 * sending its callstack info.
+	 */
+	result = pipe(pipeFileDescriptor);
+	if (0 != result) {
+		RECORD_ERROR(state, INITIALIZATION_ERROR, result);
+		goto cleanup;
+	}
+	/* Initialize pipe to be non-blocking as poll() ensures read() does not block */
```

Comment:
Please finish sentences with a period.

Reply:
I'm fine with sentence fragments, but I would like consistency:
* a sentence fragment neither starts with a capital nor ends with a period
* sentences both start with a capital and end with a period
* a comment should not be a mix of sentences and sentence fragments

I don't expect comments that are far from changes to be updated, but for a new file I think we should aim for consistency throughout.